app/medInria/medRegistrationWorkspace.cpp

```diff
@@ -257,7 +234,7 @@ void medRegistrationWorkspace::updateUserLayerClosable(int tabIndex)
 
 void medRegistrationWorkspace::updateFromRegistrationSuccess(medAbstractData *output)
```

Comment:
`medRegistrationWorkspace::updateFromRegistrationSuccess` seems not to be reached at any moment. 

It's connected by `connect(selectorToolBox(), SIGNAL(movingDataRegistered(medAbstractData*)), this, SLOT(updateFromRegistrationSuccess(medAbstractData*)));` in `medRegistrationWorkspace::medRegistrationWorkspace`. 

This emit is done in `medRegistrationSelectorToolBox::handleOutput` which seems not to be reached either. 

This function is connect through `connect(toolbox,SIGNAL(success()),this,SLOT(handleOutput()));` in `medRegistrationSelectorToolBox::changeCurrentToolBox` which is used.

Maybe you can remove all these unused functions?